Tests/Api/ClientTest.php

```diff
@@ -0,0 +1,312 @@
+<?php
+
+namespace Test\Werkspot\BingAdsApiBundle\Api;
+
+use BingAds\Bulk\ReportTimePeriod;
+use BingAds\Proxy\ClientProxy;
+use Mockery;
+use Symfony\Component\Filesystem\Filesystem;
+use Werkspot\BingAdsApiBundle\Api\Client;
+use Werkspot\BingAdsApiBundle\Api\Exceptions;
+use Werkspot\BingAdsApiBundle\Guzzle\RequestNewAccessToken;
```

Comment:
The name of this class is a bit weird 'new RequestNewAccessToken()->get()` I think i'd name it something llke `OauthTokenService->refreshToken($clientId, $clientSecret, $redirectUri, $refreshToken)` instead